### PR TITLE
Ft disable event 101389618

### PR DESCRIPTION
**Pull request by andela-aadepoju**

### CODE REVIEW

File: app/assets/javascripts/events1.js [hunk 1]
```diff
@@ -13,16 +13,28 @@ $(function(){
...
+  $(link).closest("li").hide();
 }
 
 function add_fields(link, association, content){
   var new_id = new Date().getTime();
-  var regexp = new RegExp("new_" + association, 'g');
+  var regexp = new RegExp("new_" + association, "g");
   field = content.replace(regexp, new_id);
-  $('div.event_ticket_field').append(field);
+  $("div.event_ticket_field").append(field);
```
Comment (andela-fsenjobi) [inline app/assets/javascripts/events1.js]: line 22 is missing a `var`

Comment (andela-sdamian) [inline app/assets/javascripts/events1.js]: var keyword is missing @line 22 :arrow_up:

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -48,12 +49,41 @@ $(document).ready(function() {
...
-  //this creates an animation for the scroll button at the bottom of the parallax
+}
+    to_picker.on('set', function(event) {
+        if ( event.select ) {
+            from_picker.set('max', to_picker.get('select'))
+        }
+        else if ( 'clear' in event ) {
+            from_picker.set('max', false)
+        }
+    })
```
Comment (andela-fsenjobi) [inline app/assets/javascripts/application.js]: 76, 79, 81: Missing semi colons

File: app/assets/javascripts/event_staff.js [hunk 3]
```diff
@@ -1,23 +1,94 @@
...
+            var current_field = requiredFields[i];
+            if(current_field.value == ""){
+                current_field.name = current_field.name.replace("[", " ");
+                current_field.name = current_field.name.replace("]", " ");
+                current_field.name = current_field.name.replace("_id", "");
+                empty_fields.push(current_field.name);
+            }
+        }
+        empty_fields.forEach(function(value){
+           notify(value + " is required!")
```
Comment (andela-sdamian) [inline app/assets/javascripts/event_staff.js]: Line 20 is missing a semicolon

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -48,12 +49,41 @@ $(document).ready(function() {
...
+  from_picker.on('set', function(event) {
+      if ( event.select ) {
+          to_picker.set('min', from_picker.get('select'))
+      }
+      else if ( 'clear' in event ) {
+          to_picker.set('min', false)
+      }
+  })
+} catch (e) {
+  return null
```
Comment (andela-sdamian) [inline app/assets/javascripts/application.js]: Line 64, 67 and 71 is missing a semi colon

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -48,12 +49,41 @@ $(document).ready(function() {
...
+// When something is selected, update the “from” and “to” limits.
+try {
+  from_picker.on('set', function(event) {
+      if ( event.select ) {
+          to_picker.set('min', from_picker.get('select'))
+      }
+      else if ( 'clear' in event ) {
+          to_picker.set('min', false)
+      }
+  })
```
Comment (andela-fsenjobi) [inline app/assets/javascripts/application.js]: Interesting error trapping here!

Comment (andela-sdamian) [inline app/assets/javascripts/application.js]: Line 64 and 67 missing semicolons

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -48,12 +49,41 @@ $(document).ready(function() {
...
+        to_picker = to_$input.pickadate('picker');
+
+// When something is selected, update the “from” and “to” limits.
+try {
+  from_picker.on('set', function(event) {
+      if ( event.select ) {
+          to_picker.set('min', from_picker.get('select'))
+      }
+      else if ( 'clear' in event ) {
+          to_picker.set('min', false)
```
Comment (andela-sdamian) [inline app/assets/javascripts/application.js]: Line 64 and 67 is missing a semicolon

File: app/assets/javascripts/event_staff.js [hunk 7]
```diff
@@ -1,23 +1,94 @@
...
     }
+    return isOnPage;
+  };
+
+  var generateHtml = function(data){
+    return [ "<div class='chip' data-id =",  data.user_id,  ">",
+            "<img src=", data.profile_url,  "alt='Contact Person'>",  data.first_name,  " ( ",  data.user_role,  " )&emsp;",
+             "<a href='#' data-remote='true'><span class='remove_staff'>x</span></a>",
+             "<input type='hidden' class='uid'name = 'event[event_staffs_attributes][][user_id]' value =", data.user_id, "/>",
+             "<input type='hidden' name = 'event[event_staffs_attributes][][role]' value =", data.role, "/></div>" ].join('\n');
```
Comment (andela-fsenjobi) [inline app/assets/javascripts/event_staff.js]: Seems the array style and the backtick for multiline concatenation produced the same houndci comments

File: app/assets/javascripts/application.js [hunk 8]
```diff
@@ -48,12 +49,41 @@ $(document).ready(function() {
...
+        from_picker = from_$input.pickadate('picker');
+
+    var to_$input = $('#event_end_date').pickadate(),
+        to_picker = to_$input.pickadate('picker');
+
+// When something is selected, update the “from” and “to” limits.
+try {
+  from_picker.on('set', function(event) {
+      if ( event.select ) {
+          to_picker.set('min', from_picker.get('select'))
```
Comment (andela-sdamian) [inline app/assets/javascripts/application.js]: Line 64 is missing a semicolon

**Comment by andela-iokonkwo** [comment]
I will be closing this PR pending when all the issues has been fixed.

**Pull request closed without merging**